web/index.php

```diff
@@ -33,7 +33,7 @@
 
 // Enable/Disable logging
 if (\Xibo\Helper\Environment::isDevMode() || \Xibo\Helper\Environment::isForceDebugging()) {
-    error_reporting(E_ALL);
+    error_reporting(E_ALL & ~E_DEPRECATED & ~E_USER_DEPRECATED);
```

Comment:
We don’t want to commit this change, you can report an issue related to PHP deprecated warnings.